instana/resource-synthetic-monitor.go

```diff
@@ -0,0 +1,251 @@
+package instana
+
+import (
+	"github.com/gessnerfl/terraform-provider-instana/instana/restapi"
+	"github.com/gessnerfl/terraform-provider-instana/utils"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+// ResourceInstanaSyntheticMonitor the name of the terraform-provider-instana resource to manage synthetic monitors
+const ResourceInstanaSyntheticMonitor = "instana_synthetic_monitor"
+
+const (
```

Comment:
I added which fields are currently supported in the docs for each of the different synthetic types. Since the API is still beta, only HTTPScript and HTTPAction are supported. 

https://github.com/gessnerfl/terraform-provider-instana/pull/147/files#diff-a7a3d323f7a2c6d0b9d736d13cf73e59c08373f2a30866ec9debb69cb93290bf

I haven't tested/used the other fields yet but I will need to soon. My idea was to add the additional fields in another PR once the API stabilized a bit.

Reply:
Cross that. I added a bunch of other attributes. The only ones that are missing are:
- `expectJson` 
- `expectExists`- 
- `expectNotEmpty` 
- `scripts` 

I could not manage to get the API to work with this attributes. My previous suggestion to skip them until the API stabilizes still stands.